src/app/static/src/app/generated.d.ts

```diff
@@ -171,6 +171,7 @@ export interface Filter {
 export interface FilterOption {
   label: string;
   id: string;
+  children?: any[];
```

Comment:
I couldn't find a way to make the optional children prop more specific. I tried FilterOption[] and Array<FilterOption> but TS complained for both.

Reply:
Sorry, I'm not sure why you were trying to do this? These are the generated types (auto-generated from the hintr interface) so you shouldn't be changing them manually anyway. Children is on the NestedFilterOption type.

Reply:
I changed the countryAreaFilterOption type to NestedFilterOption and this has fixed the problem.